.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,22 @@
 
 <!-- Describe your changes below in a reasonable amount of detail -->
 
+### Release
+
+Semantic release is enabled for this repository. Make sure you follow the right commit message convention. 
+
+We're using semantic-release's default — [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines):
+
+* **feat**: A new feature
+* **fix**: A bug fix
+* **docs**: Documentation only changes
+* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
+  semi-colons, etc)
+* **refactor**: A code change that neither fixes a bug nor adds a feature
+* **perf**: A code change that improves performance
+* **test**: Adding missing or correcting existing tests
+* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
+  generation
```

Comment:
What category would package updates via greenkeeper fall under?

Reply:
Good question. I'm guessing it depends on the case — maybe `fix` if the dependency had a bug that could affect poet-js, maybe `chore` if we don't want a new release at all for it (which I think will be most of the cases).